packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -4950,6 +4950,82 @@ addEventListener('fetch', event => {});`
 			await runWrangler("deploy");
 		});
 
+		it("serve_directly correctly overrides default if set to false", async () => {
```

Comment:
I know it's tested in all the other blocks implicitly, but might be nice to have an explicit check for undefined config being *omitted* in the request body. If it ever regressed and defaulted to false, then that would be a bug.

Reply:
I actually did have a test for it and then deleted after thinking "oh it's tested everywhere else" 😆  no problem, will add an explicit one!